spec/index.html

```diff
@@ -42,7 +42,7 @@
     // only "name" is required
     formerEditors:  [
       { name:       "Manu Sporny",
-        url:        "http://manu.sporny.org/",
+        url:        "https://github.com/msporny",
```

Comment:
```suggestion
        url:        "https://www.linkedin.com/in/manusporny/",
```

Reply:
Sorry, pulled the trigger too quickly; can you do a new PR?

Reply:
Fixed in https://github.com/w3c/rdf-canon/commit/0503facfaa0825686afc1f533f487816de54d9b7.